### ISSUE TITLE
Combobox placeholder contrast is too low for accessibility

### ISSUE THREAD

**Issue by juanjosegutierrez** [vmware/clarity#6439]
## Describe the bug

In the combobox component, the placeholder text color and background have a color contrast that is too low according to the Web Content Accessibility Guidelines 2.1 and my accessibility team. The contrast is currently 3 to 1 and the minimum acceptable is 4.5 to 1. Also the opacity of the placeholder text is well below 1 so it makes the placeholder text contrast even worse.

## How to reproduce

https://stackblitz.com/edit/clarity-light-theme-clr12-cds5-nrt2as?file=src/main.ts
Here is a screenshot in case the stackblitz doesn't load:
<img width="454" alt="Screen Shot 2021-10-28 at 12 45 57 PM" src="https://user-images.githubusercontent.com/19143833/139325074-811c7a56-9af4-4857-934a-eecab8e8cb90.png">

1.  Go to the stackblitz above
2.  Look at the placeholder text in the combobox

## Expected behavior

According to WCAG 2.1, the minimum contrast for placeholder texts should be at least 4.5 to 1.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 12.1.4

**Device:**

- Type: MacBook
- OS: MacOS
- Browser chrome
- Version 95

## Additional notes

None

**Comment by hippee-lee** [comment]
Thanks for reporting this. I'm looking into it to see if we have encountered this before and I'll update with more info.
